sfge/src/test/java/com/salesforce/graph/ops/expander/NullApexValueConstrainerTest.java

```diff
@@ -48,7 +48,6 @@ public static Stream<Arguments> testPathsAreCollapsedBasedOnSameNullComparison()
                 Arguments.of("String", "null == x"),
                 Arguments.of("String", "x != null"),
                 Arguments.of("String", "null != x"),
-                Arguments.of("String", "x"),
```

Comment:
The change to `ApexValue.java` caused this test to fail. However, the test itself is malformed, since `if (someString) {` is invalid apex.